### ISSUE TITLE
Nested resources?

### ISSUE THREAD

**Issue by cpursley** [hasura/ra-data-hasura#9]
Is it possible to access nested resources for queries? Perhaps a way to override and specify a custom query?

For example in the react-admin tutorial, with ra-data-hasura, is it possible to get at the `address.street` field?:

```
export const UserList = props => (
    <List {...props}>
        <Datagrid rowClick="edit">
            <TextField source="id" />
            <TextField source="name" />
            <TextField source="username" />
            <EmailField source="email" />
            <TextField source="address.street" />
            <TextField source="phone" />
            <TextField source="website" />
            <TextField source="company.name" />
        </Datagrid>
    </List>
);
```

From: https://marmelab.com/react-admin/Tutorial.html#selecting-columns

**Comment by cchampou** [comment]
Hey @cpursley ! Have you tried using ReferenceField ? In my work in progress dashboard, I'm using 
```
<ReferenceField source="delivery_address_id" reference="addresses">
   <TextField source="city" />
</ReferenceField>
```
If address is a relationship field, pointing to another model, it should work. Just fill `reference` with the actual address model, and `source` with the id of the address you're looking for.

Or maybe I misunderstood your needs. 🤷‍♂

Best regards

**Comment by cpursley** [comment]
This is an alternative: [https://github.com/Steams/ra-data-hasura-graphql](https://github.com/Steams/ra-data-hasura-graphql) built on the ra-data-graphql package.

**Comment by praveenweb** [comment]
Fixed in v0.1.0. Refer to [usage](https://github.com/hasura/ra-data-hasura#example-extending-a-query-to-include-related-entities)
